### PR TITLE
MOB-70 create info section

### PR DESCRIPTION
**Pull request by lukasrakauskas**

### CODE REVIEW

File: src/app/landing-page/info-section/info-section.component.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<section class="info-section">
+    <div class="info-about">
+        <h2>What is Mobite?</h2>
+        <p>Mobite provides you a list of phones that you might like based on your preferences.</p>
+        <p>Whether you like to takes photos, play games or want a flagship killer phone, Mobite will help you make the
+            right
+            choice. So, go ahead and never worry making a wrong purchase.</p>
```
Comment (Dezash) [inline src/app/landing-page/info-section/info-section.component.html]:
```suggestion
            choice. So, go ahead and never worry about making a bad purchase again.</p>
```

File: src/app/landing-page/info-section/info-section.component.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    </div>
+    <div class="info-why">
+        <h2>Why Mobite?</h2>
+        <div class="info-reasons">
+            <div>
+                <img src="/assets/info/pinch-easy.svg" />
+                <h3>Simple and easy</h3>
+                <p>You just have to answer few simple questions based on your preference.</p>
```
Comment (Dezash) [inline src/app/landing-page/info-section/info-section.component.html]:
```suggestion
                <p>You just have to answer a few simple questions based on your preference.</p>
```

**Review by Dezash — CHANGES_REQUESTED** [review]
There are some minor grammatical errors, but other than that everything looks good to me.

File: src/app/landing-page/info-section/info-section.component.html (hunk 2 above)
Comment (lukasrakauskas) [reply src/app/landing-page/info-section/info-section.component.html]: Done

File: src/app/landing-page/info-section/info-section.component.html (hunk 1 above)
Comment (lukasrakauskas) [reply src/app/landing-page/info-section/info-section.component.html]: Done

**Review by Dezash — APPROVED** [review]
LGTM

File: src/app/landing-page/info-section/info-section.module.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { NgModule } from '@angular/core';
+import { InfoSectionComponent } from './info-section.component';
+
+@NgModule({
+  declarations: [InfoSectionComponent],
+  imports: [],
```
Comment (MPilinkus) [inline src/app/landing-page/info-section/info-section.module.ts]: Can be deleted

**Review by MPilinkus — APPROVED** [review]

**Pull request merged**